.gitignore

```diff
@@ -21,3 +21,5 @@ src/main/resources/version.txt
 # VS Code launch settings
 .vscode/launch.test.json
 .vscode/launch.json
+.vscode/settings.json
+.gitignore
```

Comment:
You should not gitignore the `.gitignore` file itself.

Reply:
Changes implemented and checks passed.